www/common/make-backup.js

```diff
@@ -330,6 +357,11 @@ define([
             sframeChan: sframeChan
         };
         var filesData = data.sharedFolderId && ctx.sf[data.sharedFolderId] ? ctx.sf[data.sharedFolderId].filesData : ctx.data.filesData;
+        var links = ctx.sf[data.sharedFolderId] && ctx.sf[data.sharedFolderId].static ? ctx.data.static && ctx.sf[data.sharedFolderId].static : ctx.data.static;
+
+        Object.keys(links).forEach(function(key) {
+            filesData[key] = links[key];
+        });
```

Comment:
You could avoid merge the data (and risking a conflict with ids) if you provide `links` as a 5th argument to `makeFolder` 5 lines below (line 369 when I'm writing this review).